samples/subsys/ipc/openamp_rsc_table/boards/imx95_evk_mimx9596_m7.overlay

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright 2025 NXP
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/ {
+	chosen {
+		/*
+		 * shared memory reserved for the inter-processor communication
+		 */
+		zephyr,ipc_shm = &shmem;
+		zephyr,ipc_rsc_table = &rsc_table;
+		zephyr,ipc = &mailbox0;
+	};
+
+	soc {
```

Comment:
`mu7` is already added in dtsi in soc section.
Just add it at the end of the file:
```
&mu7 {
	status = "okay";
};
```